Makefile

```diff
@@ -150,6 +150,8 @@ antctl-release:
 	@echo
 	@echo "==> Running unit tests <=="
 	$(GO) test -race -coverprofile=.coverage/coverage-unit.txt -covermode=atomic -cover github.com/vmware-tanzu/antrea/pkg/...
+	sed -i '/testing\/mock_.*\.go/d' .coverage/coverage-unit.txt
```

Comment:
curious about why this is needed? wouldn't codecov just ignore these results based on your changes to the configuration file (`codecov.yaml`)?

Reply:
I thought maybe we will use other tools to inspect the result.
A second thought is that maybe all tools will compare the coverage data with sources so this will be useless.
Removed.